### PR TITLE
Contact form build

### PR DESCRIPTION
**Pull request by Gregjarvez**
Hellow Suresh. most of the layout for the form component is sorta complete. I wll add the functionality later tomorrow. Review and tell me what you would want to change.

### CODE REVIEW

File: app/containers/App/index.js [hunk 1]
```diff
@@ -34,7 +35,7 @@ export function App(props) {
       <Helmet
         titleTemplate="%s - S.K. Mukhiya"
         defaultTitle="Frontend Developer"
-        meta={[{ name: 'description', content: 'Forntend Developer, Nepal' }]}
+        meta={[{ name: 'description', content: 'Frontend Developer, Nepal' }]}
```
Comment (sureshHARDIYA) [inline app/containers/App/index.js]: @Gregjarvez : Thanks for fixing this :P

File: app/containers/Contact/index.js [hunk 2]
```diff
@@ -63,237 +49,50 @@ export default class FeaturePage extends React.Component {
...
-          </Element>
-
-          <Element id="same" className="element">
-            Two links point to this
-          </Element>
-
-          <a onClick={this.scrollToTop}>To the top!</a>
-        </div>
+        <Wrapper>
+          <Header>CONTACT ME</Header>
```
Comment (sureshHARDIYA) [inline app/containers/Contact/index.js]: @TODO, the string should be translatable :)

File: app/containers/Contact/styles/colors.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+const colors = {
```
Comment (sureshHARDIYA) [inline app/containers/Contact/styles/colors.js]: Could you move this to app/styles/colors, we can use this color code over entire app. I was thinking about this to be honest :)

File: app/containers/Contact/styles/package.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "prop-types": "^15.5.10"
+  }
```
Comment (sureshHARDIYA) [inline app/containers/Contact/styles/package.json]: Separate  `package.json` ?

File: app/containers/Contact/styles/button.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
+import colors from './colors';
+
+
```
Comment (sureshHARDIYA) [inline app/containers/Contact/styles/button.js]: Please remove extra line

File: app/containers/App/index.js [hunk 6]
```diff
@@ -10,6 +10,7 @@ import React from 'react';
 import Helmet from 'react-helmet';
 import styled from 'styled-components';
 import { FormattedMessage } from 'react-intl';
+import PropType from 'prop-types';
```
Comment (sureshHARDIYA) [inline app/containers/App/index.js]: `import PropTypes from 'prop-types';` please use this.

File: app/containers/App/index.js [hunk 7]
```diff
@@ -63,7 +64,7 @@ export function App(props) {
 }
 
 App.propTypes = {
-  children: React.PropTypes.node,
+  children: PropType.node.isRequired,
```
Comment (sureshHARDIYA) [inline app/containers/App/index.js]: `PropTypes`

File: app/containers/App/index.js (hunk 6 above)
Comment (Gregjarvez) [reply app/containers/App/index.js]: Ahhhh okay Alright sure

**Comment by sureshHARDIYA** [comment]
@Gregjarvez : Really appreciate taking time in making PR to help with the project.

**Pull request merged**